<?xml version="1.0"?>

<launch>
  <arg name="publish_covariance" default="true"/>

  <param name="/use_sim_time" value="false"/>

  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser"
    args="0.15 0.0 0.0 0.0 0.0 0.0 /base_link /laser 40" />

  <node pkg="laser_scan_matcher" type="laser_scan_matcher_node"
    name="laser_scan_matcher_node" output="screen">
    <param name="map_frame" value="map"/>
    <param name="base_frame" value="base_link" />
    <param name="odom_frame" value="base_link" />
    <param name="max_iterations" value="10"/>
    <param name="do_compute_covariance" value="1"/>
  </node>
  
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find laser_scan_matcher)/launch/demo.rviz"/>

</launch>
